core/assets/bundles/bundle_ru.properties

```
@@ -1515,9 +1515,9 @@ block.payload-loader.name = Грузовой загрузчик
 block.payload-loader.description = Загружает жидкости и ресурсы в груз
 block.payload-unloader.name = Грузовой разгрузчик
 block.payload-unloader.description = Выгружает жидкости и ресурсы из груза
-block.heat-source.name = Heat Source
-block.heat-source.description = A 1x1 block that gives virtualy infinite heat.
-block.empty.name = Пусто
+block.heat-source.name = Источник тепла
+block.heat-source.description = Блок 1x1, который дает бесконечное тепло.
```

В оригинале есть "virtually", тут его нет.
```suggestion
block.heat-source.description = Блок 1x1, который дает практически бесконечное тепло.
```